lib/kitchen/driver/aws/standard_platform/rhel.rb

```diff
@@ -50,6 +50,13 @@ def self.from_image(driver, image)
               new(driver, "rhel", (Regexp.last_match || [])[1], image.architecture)
             end
           end
+
+          def sort_by_version(images)
+            # First do a normal version sort
+            super(images)
+            # Now sort again, shunning Beta releases.
+            prefer(images) { |image| !image.name.match(/_Beta-/i) }
```

Comment:
"RHEL-7.9_HVM_BETA-20200422-x86_64-0-Hourly2-GP2"
"RHEL-7.8_HVM_GA-20200225-x86_64-1-Hourly2-GP2"
"RHEL-7.4_HVM_Beta-20170518-x86_64-1-Hourly2-GP2"
Are some examples of the strings to match.